### PR TITLE
fix checked attribute for multi checkbox

### PR DESCRIPTION
**Pull request by rodsouto**
This commit makes possible to have more than one checked element in a multi-checkbox input

### CODE REVIEW

File: tests/Helper/Input/CheckboxTest.php [hunk 1]
```diff
@@ -149,4 +149,25 @@ public function testMultiCheckbox() {
                 . '<label><input type="checkbox" name="foo[]" value="maybe" /> Maybe</label>' . PHP_EOL;
         $this->assertSame($expect, $actual);
     }
+
+    public function testMultiCheckboxWithValues() {
+        $checkbox = $this->helper;
+        $actual = $checkbox(array(
+            'name' => 'foo',
+            'value' => ['yes', 'no'],
```
Comment (harikt) [inline tests/Helper/Input/CheckboxTest.php]: looks 5.3 compatibility broken.

Comment (rodsouto) [inline tests/Helper/Input/CheckboxTest.php]: fixed :)

**Comment by harikt** [comment]
Thank you for the PR.

@pmjones in-case if you have any comments, else good to merge.

Thank you.

**Comment by pmjones** [comment]
Nice -- thanks!

**Pull request merged**